app/models/topic_list.rb

```diff
@@ -97,12 +97,16 @@ def load_topics
 
     # Create a lookup for all the user ids we need
     user_ids = []
+    group_ids = []
     @topics.each do |ft|
       user_ids << ft.user_id << ft.last_post_user_id << ft.featured_user_ids << ft.allowed_user_ids
+      group_ids << ft.allowed_group_ids
```

Comment:
Alternatively, you can do `|=` to "merge" two arrays

```suggestion
      group_ids |= ft.allowed_group_ids
```

It will automatically ensure there are no duplicates as well 👍